snmp/datadog_checks/snmp/data/default_profiles/ibm-datapower-gateway.yaml

```diff
@@ -0,0 +1,274 @@
+      OID: 1.3.6.1.4.1.14685.3.1.55.1.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorscpu1Temp
+      OID: 1.3.6.1.4.1.14685.3.1.55.2.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorscpu2Temp
+      OID: 1.3.6.1.4.1.14685.3.1.55.3.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorscpu1rpm
+      OID: 1.3.6.1.4.1.14685.3.1.55.4.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorscpu2rpm
+      OID: 1.3.6.1.4.1.14685.3.1.55.5.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorschassis1rpm
+      OID: 1.3.6.1.4.1.14685.3.1.55.6.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorschassis2rpm
+      OID: 1.3.6.1.4.1.14685.3.1.55.7.0
+  - MIB: DATAPOWER-STATUS-MIB
+    symbol:
+      name: ibm.dpStatusEnvironmentalSensorschassis3rpm
+      OID: 1.3.6.1.4.1.14685.3.1.55.8.0
+#  - MIB: DATAPOWER-STATUS-MIB
+#    symbol:
+#      name: ibm.dpStatusEnvironmentalSensorsCaseOpen
+#      OID: 1.3.6.1.4.1.14685.3.1.55.9.0
+#      enum:
+#        yes: 1
+#        no: 2
+#      TODO: enum in scalar metric is not supported yet (keep this metric and this
+#        comment in profile until it's fixed)
+#  - MIB: DATAPOWER-STATUS-MIB
+#    symbol:
+#      name: ibm.dpStatusEnvironmentalSensorsPowerSupply
+#      OID: 1.3.6.1.4.1.14685.3.1.55.13.0
+#      enum:
+#        ok: 1
+#        power-1-failure: 2
+#        power-2-failure: 3
+#        power-1-ac-lost: 4
+#        power-2-ac-lost: 5
+#      TODO: enum in scalar metric is not supported yet (keep this metric and this
+#        comment in profile until it's fixed)
+  - MIB: DATAPOWER-STATUS-MIB
+    table:
+      name: dpStatusLogTargetStatusTable
+      OID: 1.3.6.1.4.1.14685.3.1.28
+    symbols:
+      - name: ibm.dpStatusLogTargetStatusEventsProcessed
+        OID: 1.3.6.1.4.1.14685.3.1.28.1.3
+        metric_type: monotonic_count
+      - name: ibm.dpStatusLogTargetStatusEventsDropped
+        OID: 1.3.6.1.4.1.14685.3.1.28.1.4
+        metric_type: monotonic_count
+      - name: ibm.dpStatusLogTargetStatusEventsPending
+        OID: 1.3.6.1.4.1.14685.3.1.28.1.5
+        metric_type: monotonic_count
+    metric_tags:
+      - column:
+          name: ibm.dpStatusLogTargetStatusLogTarget
+          OID: 1.3.6.1.4.1.14685.3.1.28.1.1
+        tag: ibm_dp_status_log_target_status_log_target
+      - column:
+          OID: 1.3.6.1.4.1.14685.3.1.28.1.2
+          name: ibm.dpStatusLogTargetStatusStatus
+        tag: ibm_dp_status_log_target_status_status
+        mapping:
+          1: active
+          2: warning
+          3: failure
+          4: suspended
+      - column:
+          OID: 1.3.6.1.4.1.14685.3.1.28.1.6
+          name: ibm.dpStatusLogTargetStatusErrorInfo
+        tag: ibm_dp_status_log_target_status_error_info
```

Comment:
Should we keep this tag ? Afraid it will create lot of metric contexts

Reply:
Indeed, I think it's safer to comment it for now, and add it later if we think it doesn't lead to high cardinality.

note: it can be helpful to add a comment with reason why it's commented e.g. the value might be high cardinality

Reply:
@AlexandreYang Commented the tag 👍